source/user/adc.rst

```diff
@@ -6,92 +6,172 @@ Algebraic diagrammatic construction (ADC) scheme
 
 *Modules*: :mod:`adc`
 
-Background: Propagator Theory
+Introduction
 =============================
+The algebraic diagrammatic construction theory (ADC(n)) is a post-Hartree-Fock
+method used for computing correlated excited states of molecules.
+:cite:`Schirmer1983,Schirmer1998`
+The ADC methods involve a perturbative expansion of a propagator followed
+by truncation at a certain order :math:`n` that defines the ADC(n) approximation.
+The ADC(n) methods provide access to the
+excitation energies and their corresponding transition intensities via a
+'one-shot' calculation. At present, the `adc` module in PySCF contains different
```

Comment:
This is already a method-specific statement, right: it only applies to one propagator